enhancements/sdk-migrate-bundle.md

```diff
@@ -0,0 +1,362 @@
+
+In order to use this function, we will obtain the required parameters from the
+listed *source*:
+
+| parameter | source |
+| --------- | ------ |
+| directory | packagemanifest directory |
+| outputDir | default `bundle` dir or flag `--output-dir` |
+| packageName | packageName from `*.package.yaml` |
+| channels | Any channel where `channels[].currentCSV == packagedCSV.Name` from `*.package.yaml` |
+| channelDefault | defaultChannel from `*.package.yaml` or `stable` |
+
+#### Creating bundle image
+
+If we choose to build bundle images, then we can leverage the `BuildBundleImage`
+function from `opm`. This takes in two arguments:
+
+- imageTag string - tag to use for your container image
+- imageBuilder string - the container image builder i.e. `docker`, `podman`, etc
+
+This would output a new bundle image.
+
+In order to use this function, we will obtain the required parameters from the
+listed *source*:
+
+| parameter | source |
+| --------- | ------ |
+| imageTag | version of packagemanifest |
+| imageBuilder | `docker` |
+
+#### Command Line Interface
+
+The new command line will look as follows:
+
+```
+operator-sdk pkgman-to-bundle <packagemanifestdir> [--build-image=] \
+    [--output-dir=]
+```
+
+- `<packagemanifestdir>` is a positional argument that specifies the
+  packagemanifest directory.
+- `[--build-image=]` is an optional flag that indicates whether we want to build
+  the bundle as an image or just output to a directory, defaults to `bundle`
+  directory.
+- `[--output-dir=]` is an optional flag that indicates the directory to write the
+  bundle to, defaults to `bundle` directory.
+
+#### Assumptions
+
+- Existing packagemanifest directory
+
+#### Example UX Scenarios
+
+For each of the scenarios below, assume we have a packagemanifest dir with the
+following layout.
+
+```
+manifests
+└── etcd
+    ├── 0.6.1
+    │   ├── etcdcluster.crd.yaml
+    │   └── etcdoperator.clusterserviceversion.yaml
+    ├── 0.9.0
+    │   ├── etcdbackup.crd.yaml
+    │   ├── etcdcluster.crd.yaml
+    │   ├── etcdoperator.v0.9.0.clusterserviceversion.yaml
+    │   └── etcdrestore.crd.yaml
+    ├── 0.9.2
+    │   ├── etcdbackup.crd.yaml
+    │   ├── etcdcluster.crd.yaml
+    │   ├── etcdoperator.v0.9.2.clusterserviceversion.yaml
+    │   └── etcdrestore.crd.yaml
+    └── etcd.package.yaml
+```
+
+#### Simple Example
+
+Assuming the packagemanifest above, let's take the simplest example:
+
+```
+operator-sdk pkgman-to-bundle manifests
+```
+
+The command will generate 3 bundles in the default `bundle` directory:
+
+```
+bundle
+├── bundle-0.6.1
+│   ├── manifests
+│   │   ├── etcdoperator.clusterserviceversion.yaml
+│   │   ├── etcdoperator-controller-manager-metrics-service_v1_service.yaml
+│   │   ├── etcdoperator-metrics-reader_rbac.authorization.k8s.io_v1beta1_clusterrole.yaml
+│   │   └── etcdcluster.crd.yaml
+│   └── metadata
+│       └── annotations.yaml
+├── bundle-0.6.1.Dockerfile
```

Comment:
Naming these simply `bundle.Dockerfile` is fine because the parent directory name is unique,
and makes the transition to a single bundle directory for the latest version easier.
```suggestion
├── bundle.Dockerfile
```